bookmarks/services/website_loader.py

```diff
@@ -33,5 +33,8 @@ def load_website_metadata(url: str):
 
 
 def load_page(url: str):
-    r = requests.get(url)
-    return r.text
+    try:
+        r = requests.get(url, timeout=3.0)
+    except:
+        return ''
```

Comment:
Not sure if it's an improvement to add this fallback, it looks like the site was loaded but was empty. The current error handling is probably not optimal, but on the other hand, why not just let the error bubble up? It will still be handled in `load_website_metadata`.

Reply:
You are right.  try-exception block here is not necessary if timeout exception can be handled outside .